src/ontology/ro-edit.owl

```diff
@@ -6612,6 +6613,17 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0017009 "2023-03-29T17:06:06Z"
 AnnotationAssertion(oboInOwl:inSubset obo:RO_0017009 obo:RO_0002259)
 AnnotationAssertion(rdfs:label obo:RO_0017009 "is owned by"@en)
 
+# Object Property: obo:RO_0017502 (collected during)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017502 "A thorax specimen was collected during the larval stage."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017502 "An oyster was collected during low tide."@en)
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0017502 "A relation between a material entity and an occurrent in which the material entity is collected during the existence of the occurrent. The material entity is a specimen that is collected for scientific purposes."@en)
+AnnotationAssertion(terms:contributor obo:RO_0017502 <https://orcid.org/0000-0002-8844-9165>)
+AnnotationAssertion(terms:date obo:RO_0017502 "2023-09-13T04:12:32Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0017502 "collected during"@en)
+ObjectPropertyDomain(obo:RO_0017502 obo:BFO_0000040)
+ObjectPropertyRange(obo:RO_0017502 obo:BFO_0000003)
```

Comment:
general observation: this is occurrent, which is above the shoreline in COB. See #780. This comment is not to block the PR: clearly we have a precedent for using occurrent already